package.json

```diff
@@ -30,14 +28,31 @@
     "test:node-windows": "lerna run --scope \"@open-wc/rollup-plugin-html\" --scope \"@open-wc/rollup-plugin-polyfills-loader\" --scope \"polyfills-loader\" test:node",
     "test:update-snapshots": "lerna run test:update-snapshots",
     "update-dependency": "node scripts/update-dependency.js",
-    "vuepress:build": "vuepress build docs",
-    "vuepress:copy-static": "cp -f packages/demoing-storybook/demo/custom-elements.json docs/.vuepress/public/demoing/demo/custom-elements.json",
-    "vuepress:start": "vuepress dev docs"
+    "vuepress:build": "cd docs && eleventy",
+    "vuepress:serve": "es-dev-server --root-dir _site",
+    "vuepress:start": "run-p vuepress:watch vuepress:serve",
+    "vuepress:watch": "cd docs && eleventy --watch"
   },
   "dependencies": {
-    "@github/clipboard-copy-element": "^1.1.1"
+    "@fec/eleventy-plugin-remark": "^1.0.2",
```

Comment:
The linter needed some of these in `dependencies` and some in `devDependencies` and I'm not fully sure why or if it makes any sense. A later revision where we move the "docs" into a private package would likely benefit some cleanliness here.

Reply:
This move would also play well with the `@open-wc/11ty` style package that we're leaning towards here...

Reply:
just don't forget to move them over after wards. cite ☝️ this comment in the issue

Reply:
Added #1588